executor/memory_test.go

```diff
@@ -118,9 +118,7 @@ func (s *testMemoryLeak) memDiff(m1, m2 uint64) uint64 {
 }
 
 func (s *testMemoryLeak) TestGlobalMemoryTrackerOnCleanUp(c *C) {
```

Comment:
`testMemoryLeak` runs under the `SerialSuites`, it should not happen.

Reply:
We will investigate it later.